runners/katacoda/index.ts

```diff
@@ -411,4 +410,9 @@ export class Katacoda extends Runner {
         return {terminalId: this.terminalCounter, isRunning: false};
     }
 
+    private getStepsCount(runCommand: RunCommand): number {
+        let returnCount = runCommand.stepIndex == this.stepsCount - 1 ? this.stepsCount : ++this.stepsCount;
```

Comment:
I tested it with your Cobigen tutorial and with the original one. Both of them do not work inside Katacoda. There are multiple steps with the same index number in the index.json. That's why Katacoda doesn't render the steps correctly.

Reply:
oh i thought they should have the same stepindex because they are executed in the same step. I've done this, so that they are all written to the same file

Reply:
Yes, they have to be written in the same file. But then you only need one entry for this step in the index.json. At the moment there are multiple entries with the same number.